packages/webview_flutter/lib/src/callbacks.dart

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
This file is copied from [google_maps_flutter](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/lib/src/callbacks.dart).